src/ide/spies/SpyConfiguration.ts

```diff
@@ -1,25 +0,0 @@
-import { Listener } from "../../util/Notifier";
```

Comment:
I removed this file because we're not actually spying it anyway; no need to write spies for everything if we're not spying them